python_translators/translators/best_effort_translator.py

```diff
@@ -0,0 +1,85 @@
+from python_translators.translators.translator import Translator
+from python_translators.translation_query import TranslationQuery
+from python_translators.translation_response import TranslationResponse
+from python_translators.translation_costs import TranslationCosts
+from python_translators.factories.google_translator_factory import GoogleTranslatorFactory
+from python_translators.query_processors.remove_all_context import RemoveAllContext
+from python_translators.factories.microsoft_translator_factory import MicrosoftTranslatorFactory
+from python_translators.translators.glosbe_translator import GlosbeTranslator
+
+
+class BestEffortTranslator(Translator):
+
+    def __init__(self, source_language: str, target_language: str) -> None:
```

Comment:
this looks reaaaaly nice.

we have one problem however... translation has become very slow again :(  

i see two possible solutions for the current deployment: 
- start async calls to all the translators in parallel and go away as soon as three answers have returned. 
- manually check what are the fastest translators at the moment, and use a corresponding implementation here 

what do you think?